### ISSUE TITLE
Add support for Logstash 7.x

### ISSUE THREAD

**Issue by Yggdrasil** [elastic/puppet-logstash#390]
Please add support for Logstash 7.x to this module.

Given that the pcfens-filebeat module supports the 7.x branch now, and that Logstash version 7.0 has been release almost four months ago, it would be very nice to be able to upgrade our entire ELK pipeline to 7 to take advantage of the new features of that release.

**Comment by jarpy** [comment]
Logstash 7 is actually already supported. You can just do this:

```puppet
class { 'elastic_stack::repo':
  version => 7,
}

class { 'logstash':
  manage_repo => true,
}
```

You are absolutely right, though, that it should be the default now and should be clear in the documentation.

[This PR](https://github.com/elastic/puppet-elastic-stack/pull/26) should have been  merged a long time ago, and I can only apologize for the fact that it wasn't.

**Comment by jarpy** [comment]
Fixed in elastic/puppet-elastic-stack#26.
